packages/control-property-editor/package.json

```diff
@@ -15,7 +15,7 @@
         "clean": "rimraf ./dist ./coverage *.tsbuildinfo",
         "build": "npm-run-all -l -s clean -p build:compile build:webapp",
         "build:compile": "tsc  --build ./tsconfig.json --pretty --dry",
-        "build:webapp": "node esbuild.js",
+        "build:webapp": "node esbuild.js && copyfiles  --up -1 \"./src/favicon.ico\" \"./dist\"",
```

Comment:
Shouldn't this be part of the preview middleware? We were able to avoid copy tasks before and it would be good if we didn't need to add it here either.

Reply:
used https://www.npmjs.com/package/esbuild-plugin-copy?activeTab=readme for copying asset(favicon)